aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -360,6 +375,45 @@ module std::features {
         });
     }
 
+    /// Enable and disable features *for the next epoch*.
+    ///
+    /// NOTE: when it takes effects depend on feature `RECONFIGURE_WITH_DKG`.
+    /// See `aptos_framework::aptos_governance::reconfigure()` for more details.
+    ///
+    /// Can only be called by a signer of @std.
+    public fun change_feature_flags_for_next_epoch(framework: &signer, enable: vector<u64>, disable: vector<u64>) acquires PendingFeatures, Features {
```

Comment:
we discussed to let change_feature_flags call this functions and update tests accordingly?